src/transformers/configuration_utils.py

```diff
@@ -340,7 +340,8 @@ def output_attentions(self):
 
     @output_attentions.setter
     def output_attentions(self, value):
-        if self._attn_implementation != "eager":
+        # We always allow updates with the default value, which is False
+        if (self._attn_implementation != "eager") and (value is not False):
```

Comment:
why is this AMD specific error?

Reply:
I just run it on A100, it also crashed. Not sure how it behaves on T4 though, that might be why it was flagged as AMD specific.

Reply:
ok